ingress/controllers/nginx/nginx/ratelimit/main.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ratelimit
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"k8s.io/kubernetes/pkg/apis/extensions"
+)
+
+const (
+	limitIp  = "ingress-nginx.kubernetes.io/limit-connections"
+	limitRps = "ingress-nginx.kubernetes.io/limit-rps"
```

Comment:
same comment I made on the other pr about the format of annotations.
I like the modular way we're handling the annotations, but I think we should surface them all in one common place in the docs, so users know whats available without having to cd into each dir or find/grep for the `kubernetes.io` string